sdk/keyvault/azure-security-keyvault-jca/src/main/java/com/azure/security/keyvault/jca/KeyVaultKeyStore.java

```diff
@@ -258,16 +332,35 @@ public Key engineGetKey(String alias, char[] password) {
                               .orElse(null);
     }
 
+    /**
+     * Get the entry identified by the given alias.
```

Comment:
Check whether the entry identified by the given alias contains a trusted certificate.